toolchain/diagnostics/diagnostic_consumer.cpp

```diff
@@ -41,6 +41,7 @@ auto StreamDiagnosticConsumer::HandleDiagnostic(Diagnostic diagnostic) -> void {
       *stream_ << "\n";
     }
   }
+  *stream_ << "\n";
```

Comment:
Would it make sense to suppress the blank line after the final diagnostic? I think I wouldn't expect an additional blank line there.

Reply:
I've updated this PR with a force-push to do that. The force-push is so that I don't have multiple commits editing every test to need to rebase on branch updates.